tests/test_gallery.py

```diff
@@ -191,6 +191,18 @@ def test_albums_sort(settings):
     a.sort_subdirs('title')
     assert [im.title for im in a.albums] == list(reversed(titles))
 
+    orders = [d.meta['order'] for d in a.albums]
```

Comment:
This list could be hard-coded as it is the reference to do the test. It would avoid to rely on the metadata parsing code which is used later in the sort.